src/dstore/dstore_internal.h

```diff
@@ -202,15 +204,24 @@ struct dstore_io_vec {
 	struct dstore_io_buf edbuf;
 };
 
+static bool dstore_io_vec_flags_has_data(uint64_t flags) {
```

Comment:
Make it `inline` if you are adding function definition inside header file,  as per my knowledge generally we add inline function inside header file.

Reply:
done.